README.md

```diff
@@ -23,12 +23,14 @@ room client. To run the app:
    to setup and run demo server.
 2. Clone the repo
 3. ```
-   yarn
-   yarn build
-   cd `examples/fishjam-chat`
-   yarn prebuild
-   yarn run android/ios
+      yarn
+      yarn build
+      cd `examples/fishjam-chat`
+      npx expo prebuild --clean
```

Comment:
why this change? 🤔

Reply:
What change do you mean? The `npx expo prebuild` or the removed `yarn build` step?

Reply:
changing `yarn prebuild` to `npx expo prebuild`

Reply:
It's discussed as a "NOTE" below:
> Be sure to run `npx expo prebuild --clean` and not `yarn prebuild --clean` as there's an issue with path generation for ios/.xcode.env.local